torch/csrc/api/include/torch/nn/modules/pixelshuffle.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <torch/nn/cloneable.h>
+#include <torch/nn/functional/pixelshuffle.h>
+#include <torch/nn/options/pixelshuffle.h>
+
+#include <torch/csrc/WindowsTorchApiMacro.h>
+
+namespace torch {
+namespace nn {
+
+/// Rearranges elements in a tensor of shape :math:`(*, C \times r^2, H, W)`
+/// to a tensor of shape :math:`(*, C, H \times r, W \times r)`.
+/// This is useful for implementing efficient sub-pixel convolution
+/// with a stride of :math:`1/r`.
+/// Look at the paper:
+/// `Real-Time Single Image and Video Super-Resolution Using an Efficient Sub-Pixel Convolutional Neural Network`_
+/// by Shi et. al (2016) for more details.
+struct TORCH_API PixelShuffleImpl : public torch::nn::Cloneable<PixelShuffleImpl> {
+  explicit PixelShuffleImpl(const PixelShuffleOptions& options);
```

Comment:
nit: I think the argument should be named `options_`, to match the implementation